app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -92,6 +96,19 @@ private void setupInvoiceTests() {
         invoice.setBillEmail(billEmail);
     }
 
+    public void setupSchool() {
+        List<String> phoneNumbers = new ArrayList<>();
+        phoneNumbers.add("1234567890");
+        address = new Address("", "", "", "", "", "");
+        school = School.builder()
```

Comment:
both Address and School are lombok Values, which means all their fields are final, so we can just initialize them where they're being declared

Reply:
that being said, we can get rid of setupSchool altogether 
create another field the phone numbers